cypress/integration/main_page_spec.js

```diff
@@ -216,24 +217,19 @@ describe('Dashboard Home Page', () => {
       // Cypress only allows one stub per url. We make multiple POST requests to the same
       // elasticsearch endpoint. The fixture here returns a combined response of all the
       // responses for one url, effectively stubbing our elasticsearch searches.
-      cy.server();
-      cy.route('POST', '/_search/', 'fixture:elasticsearch.json').as('metrics');
-
-      cy.wait(['@metrics', '@metrics', '@metrics', '@metrics', '@metrics', '@metrics']);
-
-      cy.get('.overview-num__wrapper-home > ul#distributionErrors > :nth-child(1)').contains('6');
-      cy.get('.overview-num__wrapper-home > ul#distributionErrors > :nth-child(2)').contains('8');
-      cy.get('.overview-num__wrapper-home > ul#distributionErrors > :nth-child(3)').contains('4');
-      cy.get('.overview-num__wrapper-home > ul#distributionErrors > :nth-child(4)').contains('2');
       cy.get('.overview-num__wrapper-home > ul#distributionErrors > :nth-child(5)').contains('0');
+      cy.get('.overview-num__wrapper-home > ul#distributionErrors > :nth-child(4)').contains('2');
+      cy.get('.overview-num__wrapper-home > ul#distributionErrors > :nth-child(3)').contains('4');
+      cy.get('.overview-num__wrapper-home > ul#distributionErrors > :nth-child(2)').contains('8');
+      cy.get('.overview-num__wrapper-home > ul#distributionErrors > :nth-child(1)').contains('6');
 
-      cy.get('.overview-num__wrapper-home > ul#distributionSuccesses > :nth-child(1)').contains('7');
-      cy.get('.overview-num__wrapper-home > ul#distributionSuccesses > :nth-child(2)').contains('9');
-      cy.get('.overview-num__wrapper-home > ul#distributionSuccesses > :nth-child(3)').contains('5');
-      cy.get('.overview-num__wrapper-home > ul#distributionSuccesses > :nth-child(4)').contains('3');
       cy.get('.overview-num__wrapper-home > ul#distributionSuccesses > :nth-child(5)').contains('1');
+      cy.get('.overview-num__wrapper-home > ul#distributionSuccesses > :nth-child(4)').contains('3');
+      cy.get('.overview-num__wrapper-home > ul#distributionSuccesses > :nth-child(3)').contains('5');
+      cy.get('.overview-num__wrapper-home > ul#distributionSuccesses > :nth-child(2)').contains('9');
+      cy.get('.overview-num__wrapper-home > ul#distributionSuccesses > :nth-child(1)').contains('7');
 
-      cy.route('POST', 'http://example.com/_search/', 'fixture:updated_elasticsearch.json').as('new_metrics');
+      cy.route('POST', 'http://example.com/_search/', 'fixture:updated_elasticsearch.json');
```

Comment:
Do you still need this?

Reply:
yes that's the "updated" response after the datepicker changes the date